share/qtcreator/templates/wizards/sailfishos-qml-file/wizard.json

```diff
@@ -40,10 +40,10 @@
                {
                    "name": "QmlBaseType",
                    "trDisplayName": "Base type of QML document:",
-                   "type": "ComboBox",
+                   "type": "StringComboBox",
                    "data":
                    {
-                       "items": [ "Item" ]
+                       "items": "%{QmlLibrariesTypes}"
```

Comment:
I would consider changing upstream ComboBox so that it accepts string as well. IIUC now it already accepts list of strings and list of objects at least.

Reply:
Do you mean that we should change the standard `ComboBoxField` for using strings as items instead of creating custom `StringComboBoxField`?
And also what means IIUC?

Reply:
Yes. https://www.google.com/search?q=IIUC%20meaning